pom.xml

```diff
@@ -215,7 +228,10 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
-          <configuration />
+          <configuration>
+            <redirectTestOutputToFile>true</redirectTestOutputToFile>
+            <reportsDirectory>${project.build.directory}/test-reports</reportsDirectory>
+          </configuration>
```

Comment:
@petertrr WDYT?

Reply:
I wonder if we get any output from failing test or only a name. Anyway, both would be sufficient as it is usually easy to reproduce test failure locally

Reply:
Yes, we get the output if a test fails.